sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InfiniteRecursion.cs

```diff
@@ -90,7 +90,7 @@ string Prop0
 
         object Prop1
         {
-             get // Noncompliant {{Add a way to break out of this property accessor's recursion.}}
```

Comment:
if this is an accepted regression and FN, you should add a comment specifying the limitation

Reply:
Yes this is an accepted regression, as it was unintentional to fix this with previous PR, and the reason for the fix is because of the strange shape of CFG at that point.